### PR TITLE
Story 3 sort items

### PR DESCRIPTION
**Pull request by joseph-padfield**
Story 3 MVP - Sort functionality

### CODE REVIEW

**Review by Henry-Tottle — COMMENTED** [review]
I like this, can't spot anything but brain is warm

File: index.php [hunk 1]
```diff
@@ -59,17 +72,45 @@ function print_results($results)
...
+            if (in_array($_GET['sort'], $columns_to_sort)){
+                $sort = $_GET['sort'];
+            }
+        }
+        else {
+            $sort = 'title';
+        }
+
+        if (isset($_GET['sort_desc'])) {
+            if ($_GET['sort_desc'] === 'on') {
```
Comment (matt-ioa) [inline index.php:95]: Same here

File: style.css [hunk 2]
```diff
@@ -33,28 +33,72 @@ header {
...
 .dropdown:hover .dropdown_content {
     display: block;
 }
 
+.label_button_flex_row {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 0.5rem;
+}
```
Comment (matt-ioa) [inline style.css:67]: Add a blank line here to separate the CSS blocks

**Review by matt-ioa — APPROVED** [review]
Nice work, just one minor point to look at

**Pull request merged**